src/components/BookComponents/Thumbnail/Thumbnail.jsx

```diff
@@ -0,0 +1,33 @@
+import styles from "./Thumbnail.module.scss";
+
+const Thumbnail = ({ thumbnail, title, isList }) => {
+  // no img -> isList -> placeholder
+  // no img -> !isList -> hidden
+  return (
+    <>
+      <figure
+        className={
+          isList
+            ? styles.pics__list
+            : thumbnail
+            ? styles.pics
+            : styles.pics__hidden
+        }
+      >
+        {thumbnail ? (
+          <img
+            className={isList ? styles.pics__list__img : styles.pics__img}
+            src={thumbnail}
+            alt={title}
+          />
+        ) : (
+          <div className={styles.pics__list__img__placeholder}>
+            No thumbnail available
+          </div>
+        )}
```

Comment:
Another option is to use a default image rather than a div. You could put something in assets and render that if there's no image. If it's part of your app it won't affect performance much. Image does give alt text though, Both ways are fine